src/clientmap.cpp

```diff
@@ -596,11 +596,12 @@ int ClientMap::getBackgroundBrightness(float max_d, u32 daylight_factor,
 	};
 	if(z_directions[0].X < -99){
 		for(u32 i=0; i<sizeof(z_directions)/sizeof(*z_directions); i++){
+			// FIXME: assumes FOV of 72 and 16/9 aspect ratio.
```

Comment:
Best without the FIXME word? To not suggest this is broken. The rest of the comment is ok.